### DISCUSSION TITLE
Fallback fonts gone?

### DISCUSSION THREAD

**Discussion by jnarcane** [py-pdf/fpdf2#722] [Q&A]
Hi! I'm trying to add fallback fonts, as described in the Unicode section of the documentation. However, I get:

`AttributeError: 'FPDF' object has no attribute 'set_fallback_fonts'`

And indeed, looking at the source file (fpdf.py), there's no mention of "fallback" anywhere! I am using version 2.6.1.

**Comment by Lucas-C** [comment]
Hi @jnarcane!

Fallback fonts are a brand new feature of `fpdf2`:
* https://github.com/PyFPDF/fpdf2/pull/669
* https://github.com/PyFPDF/fpdf2/pull/712

It will be released in `v2.7.0`

Glad that you are interested in this feature!
If you want you can give install the code straight from the `git` branch behind https://github.com/PyFPDF/fpdf2/pull/712,
and provide some feedback as comments in this PR!

    pip install git+https://github.com/PyFPDF/fpdf2.git@font-fallback-markdown
